sklearn/linear_model/_logistic.py

```diff
@@ -1043,6 +1043,8 @@ class LogisticRegression(LinearClassifierMixin,
         only supported by the 'saga' solver. If 'none' (not supported by the
         liblinear solver), no regularization is applied.
 
+        |ControlParameter|
```

Comment:
Hm, I would say this is a (even traditional) example of a tuning parameter?

Reply:
Hmm... I suppose you're right.

I suspect that I over generated control parameters.